src/transformers/models/gpt2/configuration_gpt2.py

```diff
@@ -113,6 +113,12 @@ class GPT2Config(PretrainedConfig):
             Whether or not to use gradient checkpointing to save memory at the expense of slower backward pass.
         use_cache (:obj:`bool`, `optional`, defaults to :obj:`True`):
             Whether or not the model should return the last key/values attentions (not used by all models).
+        scale_attn_by_layer (:obj:`bool`, `optional`, defaults to :obj:`False):
+            [Mistral-GPT2] Whether to additionally scale attention weights by 1 / layer_idx.
```

Comment:
```suggestion
            Whether to additionally scale attention weights by 1 / layer_idx.
```

Think it would be nicer if we could remove the Mistral-GPT2 part as those flags since lots of people don't (yet) know Mistral-GPT2. I think we can create a doc page, model cards where we add a `Tips: ...` section mentioning that those flags should be enabled for Mistral-GPT2

Reply:
Done - we need to write the docs, but the references to Mistral have been removed.